shared/src/business/useCases/editDocketEntry/completeDocketEntryQCInteractor.ts

```diff
@@ -205,10 +207,17 @@ export const completeDocketEntryQCInteractor = async (
     user,
   });
 
+  // if CSS user completes a docket section QC item
```

Comment:
nit: was this comment intentionally to leave it?  the code below seems straight forward enough to not need this comment imo.  If this is logic we will need up doing everywhere in the app for this type of role, maybe we can move it somehow to be inside the entity; just a thought.

Reply:
we're in the process of removing this! i left that in to tell mrinalini yesterday

Reply:
i like the thought of moving it to the entity